public/options.html

```diff
@@ -30,9 +30,21 @@
           >
         </select>
       </div>
-      <div ng-if="vis.params.scaleType === 'static'" class="form-group">
+      <div ng-if="vis.params.scaleType === 'Static'" class="form-group">
         <bands bands="vis.params.scaleBands"></bands>
       </div>
+      <div ng-if="vis.params.scaleType === 'Dynamic - Linear'" class="form-group">
+        <div>
+          A legend scale which linearly grows from minimum to maximum range.
+        </div>
+      </div>
+      <div ng-if="vis.params.scaleType === 'Dynamic - Uneven'" class="form-group">
+        <div>
+          A legend scale that will create uneven ranges for the legend in an attempt
+          to split the map features uniformly across the ranges.  Useful when data is unevenly
+          distributed across the minimum - maximum range.
+        </div>
+      </div>
```

Comment:
Sure thing.  I'll add an informational tooltip that changes based on item selected